src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/JsonSourceGeneratorTests.cs

```diff
@@ -573,6 +573,42 @@ internal partial class JsonContext : JsonSerializerContext
             result.AssertContainsType("int");
         }
 
+        [Fact]
+        public static void NoErrorsWhenUsingTypesWithMultipleEqualsOperators()
+        {
+            // Types that override equals operators used to cause the generated to
```

Comment:
I wasn't sure of the convention around adding comments to describe why a test exists. I added it because It's not obvious from just the code what is going on. And it's not viable to describe it fully in the test's method name.

Reply:
I usually just add a comment saying "regression test for https://github.com/link/to/issue"

Reply:
I've changed it. I had a question about the `[ActiveIssue]` attribute as I thought that may be some kind of mechanism to attribute an issue with a test. But it seems it's not strictly 'active', for instance, the one at the top of the file links to an issue that is Closed.

Reply:
> That makes sense. Should I change it?

Yes please.

> I had a question about the [ActiveIssue] attribute...

You're right that it's not applicable to this particular case, it is meant to track tests that are being impacted by ongoing issues.